cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -6,11 +6,30 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Red Hat OpenShift Developer CLI (`odo`) is a tool for creating applications on {product-title} and Kubernetes. With `{odo-title}`, you can develop, test, debug, and deploy microservices-based applications on a Kubernetes cluster without having a deep understanding of the platform.
+Red Hat OpenShift Developer CLI (`odo`) is a tool for creating applications on {product-title} and Kubernetes. With `{odo-title}`, you can develop, test, debug, and deploy microservices-based applications on a Kubernetes cluster without having a deep understanding of the platform. `odo` abstracts the complex concepts of Kubernetes to help you focus on developing code. 
 
-`{odo-title}` follows a _create and push_ workflow. As a user, when you _create_, the information (or manifest) is stored in a configuration file. When you _push_, the corresponding resources are created on the Kubernetes cluster. All of this configuration is stored in the Kubernetes API for seamless accessibility and functionality.
+== Inner loop and outer loop
+
+`odo` focuses on _inner loop_ development, and provides tools that help you make the transition to the _outer loop_.
+
+Inner loop:: Consists of local coding, building, running, and testing the application. These are the activities performed by an individual developer.
+
+Outer loop:: Consists of code reviews, integration tests, and security and compliance. These are the processes used by the larger development team to eventually deploy your code to the cluster. 
+
+The inner loop primarily takes place on your development computer whereas the outer loop typically runs on shared servers, and is often automated with continuous integration and continuous deployment (CI/CD) pipelines.
+
+== Benefits of odo
+
+If you are an application developer wishing to deploy to Kubernetes, `odo` will help if:
+
+- You rely on frameworks such as Node.js, Java Spring Boot, and .NET.
+- Your application will run on Kubernetes-like infrastructure.
+- You do not want to learn Kubernetes and DevOps, to deploy to enterprise infrastructure.
```

Comment:
How about: You want to deploy to enterprise infrastructure though you might not have a deep knowledge of Kubernetes and DevOps.